--- Osmose-0-9-96-QT/Makefile~	2011-06-09 11:44:21.753728689 +0200
+++ Osmose-0-9-96-QT/Makefile	2011-06-09 11:46:34.184728662 +0200
@@ -176,7 +176,7 @@
 all: Makefile $(TARGET)
 
 $(TARGET): ui_Configuration.h ui_LogWindow.h $(OBJECTS)  
-	$(LINK) $(LFLAGS) -o $(TARGET) $(OBJECTS) $(OBJCOMP) $(LIBS)
+	$(LINK) $(LFLAGS) $(LDFLAGS) -o $(TARGET) $(OBJECTS) $(OBJCOMP) $(LIBS)
 
 Makefile: Osmose-0-9-96-QT.pro  /usr/share/qt4/mkspecs/linux-g++/qmake.conf /usr/share/qt4/mkspecs/common/g++.conf \
 		/usr/share/qt4/mkspecs/common/unix.conf \
